.docker/source/Dockerfile

```diff
@@ -4,35 +4,26 @@
 FROM moveit/moveit:melodic-ci-shadow-fixed
 MAINTAINER Dave Coleman dave@picknik.ai
 
-ENV CATKIN_WS=/root/ws_moveit
-RUN mkdir -p $CATKIN_WS/src
-WORKDIR $CATKIN_WS/src
-
-# Download moveit source
-RUN wstool init . && \
-    wstool merge https://raw.githubusercontent.com/ros-planning/moveit/${ROS_DISTRO}-devel/moveit.rosinstall && \
-    wstool update
-
-# Update apt-get because osrf image clears this cache. download deps
-# Note that because we're building on top of melodic-ci, there should not be any deps installed
-# unless something has changed in the source code since the other container was made
-# (they are triggered together so should only be one-build out of sync)
-RUN apt-get -qq update && \
-    apt-get -qq install -y \
-        wget && \
-    rosdep update && \
-    rosdep install -y --from-paths . --ignore-src --rosdistro ${ROS_DISTRO} --as-root=apt:false && \
-    rm -rf /var/lib/apt/lists/*
-
-# Replacing shell with bash for later docker build commands
-RUN mv /bin/sh /bin/sh-old && \
-    ln -s /bin/bash /bin/sh
-
-# Build repo
-WORKDIR $CATKIN_WS
-ENV TERM xterm
 ENV PYTHONIOENCODING UTF-8
-RUN catkin config --extend /opt/ros/$ROS_DISTRO --install --cmake-args -DCMAKE_BUILD_TYPE=Release && \
-    # Status rate is limited so that just enough info is shown to keep Docker from timing out, but not too much
-    # such that the Docker log gets too long (another form of timeout)
-    catkin build --jobs 1 --limit-status-rate 0.001 --no-notify
+
+# Perform a single RUN command only to optimize docker performance (reduce no. of layers)
+# https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#minimize-the-number-of-layers
+RUN \
+	mkdir src && \
```

Comment:
This is a regression to me... my previous approach was more standard and elegant:

-ENV CATKIN_WS=/root/ws_moveit	
-RUN mkdir -p $CATKIN_WS/src	
-WORKDIR $CATKIN_WS/src

Reply:
Just a note: if you're using `WORKDIR`, there is no need for `mkdir`: `WORKDIR` already creates directories if they don't exist.

Same comment earlier in this file.

Reply:
Thanks for this hint. Indeed, my intention was to cd to `CATKIN_WS` and then create (and remove) src within the single RUN command, which Dave's code didn't do.
Indeed `cd src` was missing. Sorry for that error.